### PR TITLE
Pop-up Window Creation

### PR DESCRIPTION
**Pull request by carlylechia**
For this milestone, my partner and I:
 - Used the DOM to manipulate the markup of my portfolio and create new modes and content;
 - Used javascript objects to store data and use related methods to obtain and use the data.
 - Built a modal and applied functionality to it such that it pops up when triggered and returns when closed.
 - Followed the project requirements provided by microverse and made sure the design was as close to the figma template as possible.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -325,7 +331,40 @@ <h3 id="legendtext">I'm always interested in hearing about new projects, so if y
...
+                <button type="button" class="modal-btn see-github">
+                  <p>
+                    See source <i class="fab fa-github"></i>
+                  </p>
+                </button>
+                
+              </div>
+          </div>
+      </div>
+  </div> -->
```
Comment (Hamzaoutdoors) [inline index.html]: - [ ] Kindly remove the comments from your code, leaving unnecessary comments in your project make it messy and unreadable for the user and future reviewer

File: index.html [hunk 2]
```diff
@@ -92,10 +93,15 @@ <h2 class="titlewords">My Recent Works</h2>
...
+            <button type="button" id="project-btn" class="probtn">See Project</button>
+            <!-- POP UP WINDOW -->
           </div>
         </div>
-        <div class="project-cards">
+        <div class="my-projects">
+          
+        </div>
+
+        <!-- <div class="project-cards">
```
Comment (Hamzaoutdoors) [inline index.html]: - [ ] Kindly remove the comments from your code, leaving unnecessary comments in your project make it messy and unreadable for the user and future reviewer

File: files.js/script.js [hunk 3]
```diff
@@ -32,3 +32,12 @@ function hideMenu () {
...
 
+// const parent = document.querySelector('.modal-parent'),
+//       btn = document.querySelector('button')
+//       X = document.querySelector('.close-icon');
+      
+// btn.addEventListener('click', show);
+
+// function show() {
+//   parent.style.display = 'block';
+// }
```
Comment (Hamzaoutdoors) [inline files.js/script.js]: - [ ] Kindly remove the comments from your code, leaving unnecessary comments in your project make it messy and unreadable for the user and future reviewer

File: files.js/modal.js [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+        <h2 class="desktophide">${projectList[projectList.length - 1].title}</h2>
+        <h2 class="mobilehide">Keeping track of hundreds  of components website</h2>
+        <button id="closeModal" class="closeModal" draggable = "true">${projectList[projectList.length - 1].closebtn}</button>
+      </div>
+      <div class="popupcontlangs">
+        <ul>
+          <li>${projectList[projectList.length - 1].html}</li>
+          <li>${projectList[projectList.length - 1].bootstrap}</li>
+          <li>${projectList[projectList.length - 1].ruby}</li>
+        </ul>
```
Comment (Hamzaoutdoors) [inline files.js/modal.js:91]: - [OPTIONAL]
 I recommend you here to use technologies (HTML, CSS ...) as an array in your `projectList` data, and then you can use `for loop` to iterate over them and keep your logic drier (DRY Principe - Don't Repeat Yourself)

File: files.js/modal.js [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+                 <ul>
+                   <li>${projectList[i].projectbootstrap}</li>
+                 </ul>
+               </li>
+               <li class="projectRuby">
+                 <ul>
+                   <li>${projectList[i].projectRuby}</li>
+                 </ul>
+               </li>
+             </ul>
```
Comment (Hamzaoutdoors) [inline files.js/modal.js:154]: - [OPTIONAL]
 The same here I recommend you to use technologies (HTML, CSS ...) as an array in your `projectList` data, and then you can use `for loop` to iterate over them and keep your logic drier (DRY Principe - Don't Repeat Yourself).

**Review by Hamzaoutdoors — CHANGES_REQUESTED** [review]
Hey team, I am **Hamza** your reviewer for this PR 😸 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

image

## To Highlight
✔️ The repo looks professional
✔️ Git/GitHub flow best practices have been followed
✔️ You have a descriptive README

## Required Changes ♻️
- [Required]
You were asked to add [JavaScript Linters](https://github.com/microverseinc/linters-config/tree/master/javascript), Kindly add this linters to your ` linters.yml` file
_See the inline comments below for more required changes 😃_  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [ ] Please try to add `.DS_Store` file in your `.gitignore` file
- [ ] Add `LICENSE MIT` to your project, as you are using the link in the README file because keeping broken links in your readme file is something Unprofessional

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 6]
```diff
@@ -39,7 +39,3 @@ Give a ⭐️ if you like this project!
 - Hat tip to anyone whose code was used
 - Inspiration
 - etc
-
-## 📝 License
-
-This project is [MIT](./MIT.md) licensed.
```
Comment (elyor-doniyorov) [inline README.md]: - [ ] Kindly add the license to your Readme file as the previous reviewer requested.

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @Christianib003, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
- [ ] Please use [html,css,Javascript linter](https://github.com/microverseinc/linters-config/tree/master/html-css-js) config for this project as requested by the previous reviewer.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: files.js/modal.js [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+  `;
+  popUp.innerHTML = showing;
+  body.appendChild(popUp);
+
+  document.addEventListener('click', (e) => {
+    if (e.target.id === 'closeModal') {
+      popUp.style.display = 'none';
+    }
+  });
+}
```
Comment (FrederickMih) [inline files.js/modal.js:116]: - [**OPTIONAL**] What do you think of making our popup properly responsive on the desktop sizes?. Let's check out how it looks on your design and Figma design
   
  

     Your Implementation | Figma Design
    :---:|:---:
     | 

- Did you notice how your buttons float over the text?. Please let's try to fix this.

**Review by FrederickMih — APPROVED** [review]
## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: files.js/modal.js (hunk 7 above)
Comment (carlylechia) [reply files.js/modal.js:116]: Thank you @FrederickMih . I have adjusted it

**Comment by ShoiraTa** [comment]
### **STATUS: APPROVED  ✔️** 
------
Hi team,  
Approving request on your dashboard

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**